pkg/webhook/nad/validator.go

```diff
@@ -48,8 +49,9 @@ func (n *nadValidator) Create(_ *types.Request, newObj runtime.Object) error {
 		return fmt.Errorf(createErr, netAttachDef.Namespace, netAttachDef.Name, err)
 	}
 
+	// The VLAN value of untagged network will be empty or number 0.
```

Comment:
I think we should limit the untagged network to empty only.

Reply:
The value of vlan field will be 0 after JSON unmarshaling even though the vlan field is empty.